app/assets/javascripts/application.js

```diff
@@ -25,6 +25,11 @@ const constants = {
   SQ_FT_CONVERT: 10.764,
   // price to be defined by Snow Queen
   PRICE_PER_SQ_FT: 0.15,
+  // sets map boundaries to Port Moody and North Coquitlam
+  DEFAULT_BOUNDARIES: {
+    TOP: [49.327512, -122.755847],
+    BOTTOM: [49.271039, -122.892113],
```

Comment:
rename to `NE`, `SW` (or `NORTH_EAST` and `SOUTH_WEST`, or even separate them into `NORTH`, `EAST`, `SOUTH`, `WEST`